l10n_it_split_payment/models/__init__.py

```diff
@@ -1,8 +1,12 @@
 # Copyright 2015  Davide Corio <davide.corio@abstract.it>
 # Copyright 2015  Lorenzo Battistini - Agile Business Group
 # Copyright 2016  Alessio Gerace - Agile Business Group
+# Copyright 2023  Giuseppe Borruso <gborruso@dinamicheaziendali.it>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from . import account
+from . import account_fiscal_position
+from . import account_move
+from . import account_move_line
+from . import comapny
```

Comment:
lo puoi rinominare in `company`? Qui e il file` .py`.

Reply:
svista grazie